### PR TITLE
[docs] fix harperdb

### PR DESCRIPTION
**Pull request by withshubh**

### CODE REVIEW

File: docs/versioned_docs/version-2.8.0/marketplace/plugins/harperdb.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ Before following this guide, it is recommended to check the following doc: **[Us
 ## Connection
 
 To establish a connection with HarperDB, you need the following credentials:
-1. **Host**: The hostname or IP address of your HarperDB instance (e.g., `https://myinstance.harperdbcloud.com`).
-2. **Port**: The port number configured for your server (default is `9255`).
+1. **Host**: The hostname or IP address of your HarperDB instance (e.g., `55.22.285.222` or `myinstance.harperdbcloud.com`).
+2. **Port**: The port number configured for your server (default is `9925`). If you are using HarperDB Studio(cloud), leave the field empty.
```
Comment (akshaysasidrn) [inline docs/versioned_docs/version-2.8.0/marketplace/plugins/harperdb.md:22]: You can mention rather port 443

File: docs/docs/marketplace/plugins/harperdb.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ Before following this guide, it is recommended to check the following doc: **[Us
 ## Connection
 
 To establish a connection with HarperDB, you need the following credentials:
-1. **Host**: The hostname or IP address of your HarperDB instance (e.g., `https://myinstance.harperdbcloud.com`).
-2. **Port**: The port number configured for your server (default is `9255`).
+1. **Host**: The hostname or IP address of your HarperDB instance (e.g., `55.22.285.222` or `myinstance.harperdbcloud.com`).
+2. **Port**: The port number configured for your server (default is `9925`). If you are using HarperDB Studio(cloud), leave the field empty.
```
Comment (withshubh) [inline docs/docs/marketplace/plugins/harperdb.md:22]:
```suggestion
1. **Host**: The hostname or IP address of your HarperDB instance (e.g., `162.156.250.74` or `myinstance.harperdbcloud.com`).
2. **Port**: The port number configured for your server (default is `9925`). If you are using HarperDB Studio(cloud), leave the field empty or set it to `443`.
```

File: docs/versioned_docs/version-2.8.0/marketplace/plugins/harperdb.md (hunk 1 above)
Comment (withshubh) [inline docs/versioned_docs/version-2.8.0/marketplace/plugins/harperdb.md:22]:
```suggestion
1. **Host**: The hostname or IP address of your HarperDB instance (e.g., `162.156.250.74` or `myinstance.harperdbcloud.com`).
2. **Port**: The port number configured for your server (default is `9925`). If you are using HarperDB Studio(cloud), leave the field empty or set it to `443`.
```

Comment (withshubh) [reply docs/versioned_docs/version-2.8.0/marketplace/plugins/harperdb.md]: Updated

**Review by akshaysasidrn — APPROVED** [review]

**Pull request merged**